sources/packages/web/src/assets/css/base.scss

```diff
@@ -238,3 +238,8 @@ html * {
 .container-max-width {
   max-width: 730px !important;
 }
+
+// Btn front color white.
+.btn-font-color-light .v-btn__content {
```

Comment:
What is the purpose of this class?

Reply:
did you mean style? 
![image](https://user-images.githubusercontent.com/77353155/188737467-18238387-34bc-40cc-b3de-14bb6f1be8a0.png)
The font color of the btn was black previously.